lib/ffmpeg/encoderbase.py

```diff
@@ -33,6 +33,7 @@ class Encoder(PropertyHandler, BaseFormatMapper):
   encoded       = property(lambda s: s._encoded)
   osencoded     = property(lambda s: filepath2os(s.encoded))
   encoded_ext   = property(lambda s: s.props["encoded_ext"])
+  skip_checks   = property(lambda s: s.props["skip_checks"])
```

Comment:
This needs to be an optional property.  If not specified, it should return an empty list.  Otherwise, we are going to break other test cases (as seen in CI pre-check results).